benchmarks/dlrm/ootb/dlrm_s_pytorch.py

```diff
@@ -2361,8 +2368,8 @@ def run():
                                     },
                                 )
                             break
-                if k == 0 and args.fb5logger is not None:
-                    fb5logger.run_stop(nbatches - args.warmup_steps, args.mini_batch_size)
+                if k == 0:
```

Comment:
Are we only capturing the first epoch?

Reply:
Looks like it. I did not write this personally but seems like we always run 1 epoch so it isn't an issue. In the run commands we do not set a value for args.nepochs and the default is 1. Might be better to change this to log for any number of epochs in another PR.